src/core/oned/ITFReader.ts

```diff
@@ -91,7 +91,7 @@ export default class ITFReader extends OneDReader {
      *!/
     */
 
-    public decodeRow(rowNumber: number, row: BitArray, hints?: Map<DecodeHintType, any>): Result {
+    public decodeRow(rowNumber: number, row: BitArray, startGuardRange: number[], hints?: Map<DecodeHintType, any>): Result {
```

Comment:
Can I ask you why you added this startGuardRange do OneD readers?

Just asking because I couldn't find'em in the ZXing sources: https://github.com/zxing/zxing/blob/0dfda0807876a53be295fa0abafd53866edf13bd/core/src/main/java/com/google/zxing/oned/Code128Reader.java#L236

Reply:
You're right! I did a refactoring and all the tests continue to pass

Reply:
Thanks! I didn't see a real problem there, the only reason I asked that is because we wanna make the code look the closest as possible with the original sources, so it's accessible to future contributions like yours; and there's even things we need to improve in our side.